plugins/activity-resources/src/components/utils.ts

```diff
@@ -37,14 +37,21 @@ async function createPseudoViewlet (
   // Check if it is attached doc and collection have title override.
   const presenter = await getObjectPresenter(client, doc._class, { key: 'doc-presenter' })
   if (presenter !== undefined) {
+    let collection = ''
+    if (dtx.collectionAttribute?.label !== undefined) {
+      if (dtx.collectionAttribute.isCustom) {
```

Comment:
If we have embedded why we need this logic? And it looks wrong

Reply:
It's outdated, Thanks